src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+import java.util.Scanner;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String name = "Brad";
+        System.out.println("Hello I am " + name + ".\n");
+        System.out.println("How can I help you today?\n");
+        Scanner userInput = new Scanner(System.in);
+        List inputlist = new List();
+        while (true) {
+            String x = userInput.nextLine();
+            boolean canExit = x.equals("bye");
+            boolean canList = x.equals("list");
+            boolean canMark = x.startsWith("mark");
+            boolean canUnmark = x.startsWith("unmark");
+            if (canExit) {
+                printOutput("Bye. Hope to see you again soon!");
+                break;
+            } else if (canList) {
+                printOutput("Here are the tasks in your list:\n" + inputlist.getList());
+            } else if (canMark) {
+                int taskNum = Integer.parseInt(x.split(" ")[1]);
+                if (taskNum > inputlist.listSize()) {
+                    System.out.println("Exceeded existing list size of: " + inputlist.listSize() + ". Please enter a valid number" + "\n");
+                } else {
+                    inputlist.markAsDone(taskNum, true);
+                    String message = "Nice! I've marked this task as done: \n" + inputlist.getTask(taskNum);
```

Comment:
This line is too long (88 characters). Try to keep your lines within 80 characters. You could do it this way:
```
String message = "Nice! I've marked this task as done: \n" +
        inputlist.getTask(taskNum);
```